CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Release Notes
 
+## Version 1.9.0- 2020-03-26
+
+New features & improvements
+
+* Update DefaultKubernetesVersion to v1.18.0 [#7235](https://github.com/kubernetes/minikube/pull/7235)
+* Add --vm flag for users who want to autoselect only VM's [#7068](https://github.com/kubernetes/minikube/pull/7068)
+* Add 'stable' and 'latest' as valid kubernetes-version values [#7212](https://github.com/kubernetes/minikube/pull/7212)
+
+* gpu addon: privileged mode no longer required [#7149](https://github.com/kubernetes/minikube/pull/7149)
+* Add sch_tbf and extend filter ipset kernel module for bandwidth shaping [#7255](https://github.com/kubernetes/minikube/pull/7255)
+* Parse --disk-size and --memory sizes with binary suffixes [#7206](https://github.com/kubernetes/minikube/pull/7206)
+* local search for the site [#7253](https://github.com/kubernetes/minikube/pull/7253)
```

Comment:
this last item can be removed as documentation isn't included in the release binary